docs/components/query.md

```diff
@@ -0,0 +1,26 @@
+# Query
+
+The query component implements the Prometheus HTTP v1 API to query data in a Thanos cluster via PromQL.
+It joins a Thanos cluster mesh and can access all data exposed by store APIs within it. It is fully stateless and horizontally scalable.
+
+The querier needs to be passed a list of peers to through which it can join the Thanos cluster.
+This can either be a repeated list of peer addresses or a DNS name, which it will resolve for peer IP addresses.
+
+```
+$ thanos query \
+    --http-address     "0.0.0.0:9090" \
+    --cluster.peers    "thanos-cluster.example.org" \
+```
+
+The query layer can deduplicate series that were collected from high-availability pairs of data sources such as Prometheus.
+A fixed replica label must be chosen for the entire cluster and can then be passed to query nodes on startup.
+Two or more series that have that are only distinguished by the given replica label, will be merged into a single time series. This also hides gaps in collection of a single data source.
```

Comment:
`s/will be merged into a single time series./will be merged into a single time series by default./`

Reply:
Is that actually true? Don't you have to opt-in via setting the param in the HTTP request?

Reply:
ah sorry I was thinking about deduplication here!